### PR TITLE
Add a fileHandler into logger to write the logs to a file

### PR DESCRIPTION
**Pull request by workingloong**
Except outputing the logs to stdout, we also need to write the logs to a file. Then, the log system can collect logs from the file.

### CODE REVIEW

File: elasticdl/python/common/log_utils.py [hunk 1]
```diff
@@ -2,29 +2,50 @@
 import sys
 import typing
 
-_DEFAULT_LOGGER = "elastic.logger"
+LOG_FILE = "/home/admin/logs/elasticdl.log"
```
Comment (terrytangyuan) [inline elasticdl/python/common/log_utils.py:5]: This is only internal path. Others may not have this dir.

File: elasticdl/python/worker/worker.py [hunk 2]
```diff
@@ -21,7 +21,10 @@
     scatter_embedding_vector,
     string_to_id,
 )
-from elasticdl.python.common.log_utils import get_logger
+from elasticdl.python.common.log_utils import (
+    get_logger,
+    LOG_FILE,
```
Comment (terrytangyuan) [inline elasticdl/python/worker/worker.py:26]: You can just use `LOG_FILE` inside `log_utils` and provide it as the default value for argument `filename`.

File: elasticdl/python/common/log_utils.py [hunk 3]
```diff
@@ -2,29 +2,50 @@
...
+def get_file_hander(filename):
+    file_handler = logging.FileHandler(
+        filename, mode='a', encoding="UTF-8", delay=True
+    )
+    file_handler.setFormatter(_DEFAULT_FORMATTER)
+    return file_handler
+
+
+def get_logger(
+    name, level="INFO", filename=None, handlers=None, update=False
```
Comment (terrytangyuan) [inline elasticdl/python/common/log_utils.py:31]: filename -> file_name

**Comment by workingloong** [comment]
The file handler cannot redirect the trace stack into the file. We need to use the redirection command in Linux like ">> log.txt 2>&1". So, I close the PR.

**Pull request closed without merging**